_pytest/config.py

```diff
@@ -1136,7 +1140,11 @@ def determine_setup(inifile, args):
                 if rootdir.join("setup.py").exists():
                     break
             else:
-                rootdir = ancestor
+                dirs = get_dirs_from_args(args)
```

Comment:
I don't think we need this actually. If we pass `args` directly into `getcfg`, they are validated as valid paths